cosmwasm/ucs03-zkgm-api/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name    = "ucs03-zkgm-api"
+version = "0.0.0"
+
+authors      = { workspace = true }
+edition      = { workspace = true }
+license-file = { workspace = true }
+publish      = { workspace = true }
+repository   = { workspace = true }
+
+[lints]
+workspace = true
+
+[dependencies]
+alloy-primitives  = { workspace = true }
+alloy-sol-types   = { workspace = true }
+cosmwasm-schema   = { workspace = true }
+cosmwasm-std      = { workspace = true, features = ["std"] }
+enumorph          = { workspace = true }
+ibc-union-spec    = { workspace = true, features = ["schemars", "serde"] }
+token-factory-api = { workspace = true }
+unionlabs         = { workspace = true, default-features = false }
```

Comment:
default-features = false here doesn't do anything, and i think cargo may complain about it